tests/timeout/fuzz.js

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 
-/**
- * @param { Buffer } data
- */
+// Timeouts are directly handled by libFuzz and can not be intercepted.
```

Comment:
`libfuzzER`?

Reply:
Also, any reason the `@param` was removed?

Reply:
fixed. 
I don't find the `@param { Buffer } data` comments useful. Reverted it, though.